PDBminer

```diff
@@ -976,7 +950,8 @@ def align_uniprot_pdb(pdb_id, uniprot_sequence, uniprot_numbering, mut_pos,
         df_align, mutations_in_all, warnings = processing_normal_alignment(uniprot_sequence, 
                                                                            list(chain_df.residue_name), 
                                                                            uniprot_numbering_copy, 
-                                                                           list(chain_df.b_factor))
+                                                                           list(chain_df.b_factor),
```

Comment:
it's usually more robust to use the `pandas` API to turn something into a list, i.e.

```
chain_df.b_factor.to_list()
```